OpenVPNHelper/OpenVPNHelper.m

```diff
@@ -85,6 +85,43 @@ - (BOOL)verify:(NSString *)identifier atURL:(NSURL *)fileURL {
 }
 
 - (void)startOpenVPNAtURL:(NSURL *_Nonnull)launchURL withConfig:(NSURL *_Nonnull)config upScript:(NSURL *_Nullable)upScript downScript:(NSURL *_Nullable)downScript leasewatchPlist:(NSURL *_Nullable)leasewatchPlist leasewatchScript:(NSURL *_Nullable)leasewatchScript scriptOptions:(NSArray <NSString *>*_Nullable)scriptOptions reply:(void(^_Nonnull)(BOOL))reply {
+    
+    // Get the path of config file
+    NSString* path = config.path;
+    NSString* content = [NSString stringWithContentsOfFile:path
+                                                  encoding:NSUTF8StringEncoding
+                                                     error:NULL];
+    
+    NSMutableArray *listItems = [content componentsSeparatedByString:@"\r\n"];
```

Comment:
Not sure if the line endings are always \r\n?

Reply:
Yes, in open vpn config file, each we can separate by /r/n

Reply:
But is an openvpn config file with \r or with \n also a valid config file?

Reply:
Yes it is. You can have the "DOS/Windows" encoding or UNIX/Linux/macOS encoding. On UNIX/Linux/macOS you just have `\n`, on Windows you have `\r\n`.

So you need to support both.

Reply:
The current implementation would only remove lines for Windows encoding. It should be adjusted to support the other line endings too.